internal/pkg/service/common/task/node_test.go

```diff
@@ -650,7 +650,7 @@ task/123/my-receiver/my-export/some.task/%s
 								attribute.String("task_type", "some.task"),
 								attribute.Bool("is_success", false),
 								attribute.Bool("is_application_error", true),
-								attribute.String("error_type", "other"),
+								attribute.String("error_type", "insufficientStorage"),
```

Comment:
Why `unexpectedError` = `insufficientStorage`?

Reply:
Because we added support for  `errors.WithName` interface.